Sources/Playbook/Components/Loader/PBLoader.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  Playbook Swift Design System
+//
+//  Copyright © 2024 Power Home Remodeling Group
+//  This software is distributed under the ISC License
+//
+//  PBLoader.swift
+//
+
+import SwiftUI
+
+struct PBLoader: View {
+  @State private var rotationAngle: Double
+  @State private var dotIndex: Int
+  @State private var contentSize: CGSize
+  let dotsCount: Int
+  let dotSize: CGFloat
+  let circleRadius: CGFloat
+  let spinnerFont: PBFont
+  let spinnerSpeed: TimeInterval
+  let alignment: HorizontalAlignment
+  public init(
+    rotationAngle: Double = 0,
+    dotIndex: Int = 0,
+    contentSize: CGSize = .zero,
+    dotsCount: Int = 7,
```

Comment:
I believe it should be 8 dots.

Reply:
hey! I just pushed this so I could change branches. Im not done. Should have said something, sorry!